site/content/contribute/webapp/e2e-cheatsheet/main-menu.md

```diff
@@ -1,37 +1,37 @@
 ---
-title: "Main Menu"
+title: "Product Menu"
 date: 2020-12-11T00:00
 weight: 4
 subsection: Cypress cheatsheet
 ---
 
-![image](/contribute/webapp/e2e-cheatsheet/main-menu.png)
+![image](/contribute/webapp/e2e-cheatsheet/product-menu.png)
 
 ***
 
 ### `cy.uiOpenMainMenu(item)`
-Open the Main Menu by clicking the team header title or hamburger icon at the LHS.
+Open the Product menu by clicking the product name in the Global Header.
 
 - `item`
 < <a target="_blank" href="https://developer.mozilla.org/en-US/docs/Web/JavaScript/Data_structures#String_type">string</a> >
-  - Possible values: `'Account Settings'`, `'Invite People'`, `'Team Settings'`, `'Manage Members'`, `'Create a Team'`, `'Join Another Team'`, `'Leave Team'`, `'Integrations'`, `'Plugin Marketplace'`, `'System Console'`, `'Help'`, `'Keyboard Shortcuts'`, `'Report a Problem'`, `'Download Apps'`, `'About Mattermost'` and `'Logout'`.
+  - Possible values: `'Channels'`, `'Boards'`, `'Playbooks'`, `'System Console'`, `'Integrations'`, `'Marketplace'`, `'Download Apps'`, and `'About Mattermost'`.
 
-##### Open the Main Menu
+##### Open the Product menu
 ```javascript
-// # Open 'Main Menu'
+// # Open 'Product menu'
 cy.uiOpenMainMenu();
```

Comment:
@saturninoabril - Given the terminology change from "Main Menu" to "Product menu" -- can you assist with updates on this page, please? I've updated all text-based references. Can you update the code-specific references?

Reply:
Sure @cwarnermm! Please see changes made at https://github.com/mattermost/mattermost-developer-documentation/pull/972/commits/52a50fb04f97d0874d14833a3f0980c93a0fa8a3

Reply:
Thank you, @saturninoabril!